regenerate.sh

```diff
@@ -27,8 +29,16 @@ tv_scripts=(
     zip_0244
     zip_0316)
 
+formats="${1:-rust json zcash}"
+
 for generator in "${tv_scripts[@]}"
```

Comment:
While we're fixing this, something that I frequently find myself doing is just regenerating a single set of test vectors, instead of regenerating them all. Now that we no longer use `$2`, maybe if that's present we could use that argument as the sole value in `tv_scripts` instead of regenerating everything?

Reply:
The approach of using `poetry run SCRIPT_NAME` directly doesn't put the files in the expected place by itself.

Reply:
The (corrected) script takes a list of formats, so there would be a conflict unless it implemented more complicated argument parsing. You can always write `poetry run SCRIPT_NAME -t FORMAT >OUTPUT_FILE`.

Reply:
Ah, I see. Hm. I guess I'll just have to keep commenting out bits of this script when I need to regenerate stuff.